releaseNotes/MicrosoftAzureAppConfigurationFunctionsWorker.md

```diff
@@ -1,6 +1,11 @@
 # Microsoft.Azure.AppConfiguration.Functions.Worker
 [Source code][source_code] | [Package (NuGet)][package]
 
+## 7.2.0 - May 2, 2024
+### Enhancements
+* Added support for .NET 8 as a target framework. [#530](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/530)
```

Comment:
We normally just mention this in the base package release notes.

Reply:
Correct for the base, but it is a change for this package too.

Reply:
That's true. We've just been following this pattern in our previous release notes. However, I'm also fine with adding it to all release notes going forward.

Reply:
Yes, we can do it more explicitly going forward.